packages/api/package.json

```diff
@@ -15,6 +16,8 @@
   },
   "devDependencies": {
     "@atproto/lex-cli": "*",
-    "@atproto/pds": "*"
+    "@atproto/pds": "*",
+    "@npmcli/package-json": "^3.0.0",
```

Comment:
don't hate me for this, but what do you think of putting these in the monorepo root instead of each individual pacakge

Reply:
part of me also just wants to move the build script to the monorepo root as well & reuse it 🤔

Reply:
Might be easier to chat about this, but main thoughts I'm having are:
 - I lean towards being honest with dependencies, and declaring them wherever they are needed.  So if the build script lives in `api/` and needs these deps, I would lean towards keeping them here.  But if the build script using these deps lives elsewhere, I'd tend to move them there.
 - Yarn is deduping and hoisting these dependencies for us, so we shouldn't have dupes lying around.
 - Maybe a good middle-ground would be to make a package for build tooling.  That would give us a chance to centralize the deps and logic, and allow us to make exceptions for packages that need some special build treatment.
 - I just realized esbuild is installed in the root, so I created a little inconsistency here at best.

Reply:
alright cool that makes sense 👍 
Yeah my general rule of thumb was to keep honest about dependencies but group most dev dependencies in the monorepo root. Makes it easier to track & switch out without leaving cruft hanging around.
I dig the reasoning tho. I like the idea of splitting out dev tooling into it's own package eventually though. Fine with it as it for now & we can always readdress at some point

Reply:
That makes sense.  It sounds like a decent way to force you to keep all the packages in sync with each other 👍